pkg/common/aws/ec2.go

```diff
@@ -3,13 +3,20 @@ package aws
 import (
 	"fmt"
 	"strings"
+	"time"
 
 	"github.com/openshift/osde2e/pkg/common/config"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/ec2"
 )
 
+const (
+	tagKeyForExemptEC2Instances = "osde2e-proxy"
+)
+
+var ErrOutOfTerminateEC2Instance = fmt.Errorf("Unable to terminate EC2 instances: \n")
```

Comment:
```suggestion
var ErrTerminateEC2Instances = fmt.Errorf("unable to terminate EC2 instances")
```
drop the `OutOf` and treat the error string as a fragment of a larger string. It should be possible to join together errors to make a legible sentence (`failed to run osde2e cleanup: unable to terminate EC2 instances: authentication failed` for example

Reply:
Done